warehouse/api.go

```diff
@@ -772,7 +772,7 @@ func validateObjectStorage(ctx context.Context, request *ObjectStorageValidation
 	factory := &filemanager.FileManagerFactoryT{}
 	fileManager, err := factory.New(getFileManagerSettings(request.Type, request.Config))
 	if err != nil {
-		return fmt.Errorf("unable to create file manager: %s", err.Error())
+		return fmt.Errorf("unable to create file manager: \n%s", err.Error())
```

Comment:
Is the UI supposed to break the string based on \n and then render them 1 by 1 ?

Reply:
The \n divide the string internally  into 1+ lines. 1st line represent the explanation of error from our side and then rest line will be the exact error we are getting.

Reply:
\n won't automatically create line breaks in UI, I need to update the CSS (trivial change). But this will also cause more line breaks if \n is part of the actual error (for example bucket name).

Reply:
Yeap it will create one extra line, but except first line everything will be the exact error

Reply:
Also, we can have multiple line break(bucket name case you mentioned)

Reply:
I mean if the bucket name is "shashank\n", then "\n" won't show up in the error message

Reply:
Yeap printing the string won't show it up

Reply:
Hey but we are escaping the string once. That means it will show up. I am 
<img width="766" alt="Screenshot 2022-10-12 at 6 02 46 PM" src="https://user-images.githubusercontent.com/36361506/195343545-ba4092c2-226d-4286-ac98-3b77d0dd53bf.png">

Reply:
As long as you are confident that this change won't mess up other parts of the message, I'm good